charts/argocd-applicationset/values.yaml

```diff
@@ -146,3 +146,58 @@ extraVolumes: []
 # -- List of extra cli args to add
 extraArgs: []
   # - --loglevel=warn
+
+## Override APIVersions
+## If you want to template helm charts but cannot access k8s API server
+## you can set api versions here
+apiVersionOverrides:
+  # -- String to override apiVersion of ingresses rendered by this helm chart
+  ingress: "" # networking.k8s.io/v1beta1
+
+## Webhook for the Git Generator
+## Ref: https://argocd-applicationset.readthedocs.io/en/master/Generators-Git/#webhook-configuration)
+webhook:
+  ingress:
+    # -- Enable an ingress resource for Webhooks
+    enabled: false
+    # -- Additional ingress annotations
+    annotations: {}
+    # -- Additional ingress labels
+    labels: {}
+    # -- Defines which ingress controller will implement the resource
+    ingressClassName: ""
+
+    # -- List of ingress hosts
+    ## Hostnames must be provided if Ingress is enabled.
+    ## Secrets must be manually created in the namespace
+    hosts:
+      []
```

Comment:
Isn't this inconsistent, having the [] on a separate line ? Even if probably more convenient

Reply:
Yes, you are right. Copied this blindly form `argo-cd/values.yaml`.